demo/docs/quickstart/installation.md

```diff
@@ -7,32 +7,36 @@ nav_order: 1
 
 Install the plugin either as a Ruby gem or by copying the pre-built plugin file to your Jekyll site.
 
-# Install as Ruby Gem
+# Install the plugin as Ruby Gem
 
-1. Add gem to Gemfile
-```
- gem "jekyll-lunr-js-custom-search", :git => "git://github.com/dnoneill/jekyll-lunr-js-custom-search.git"
-```
+1. Add the Ruby gem to your `Gemfile`:
+  ```
+   gem "jekyll-lunr-js-custom-search", :git => "git://github.com/dnoneill/jekyll-lunr-js-custom-search.git"
+  ```
 
-2. Modify your Jekyll _config.yml file to include the Gem.
-```
-plugins:
-  - jekyll-lunr-js-custom-search
-```
+2. Add `jekyll-lunr-js-custom-search` as plugin to you your Jekyll `_config.yml` file:
+  ```
+  plugins:
+    - jekyll-lunr-js-custom-search
+  ```
 
-# Install pre-built plugin
+<!-- Result or next steps? --> 
 
-Place build/jekyll_lunr_js_custom_search.rb inside the `_plugins` folder in the root of your Jekyll site.
+# Copy pre-built plugin file
 
-Each time the site is generated, the content from all Jekyll posts and pages will be indexed to a js/index.js file ready for lunr.js to consume.
+1. Copy the `build/jekyll_lunr_js_custom_search.rb` file to the `_plugins` folder in the root of your Jekyll site.
```

Comment:
The project readme says: 

> Then copy `build/jekyll_lunr_js_search.rb` to your Jekyll site's _plugins folder 

_file does not exist - name changed?_ If so `build/jekyll_lunr_js_custom_search.rb` is correct here? And, the project's readme.md must be updated.

> and the build/*.min.js files to your site's js folder.

This is included in the readme.md but not here. Which is correct?